app/models/evaluation.rb

```diff
@@ -44,9 +44,68 @@ class Evaluation < ApplicationRecord
     }
   ]
 
+  SORTING = ['']
+
+  def self.paginate_evaluations(json=nil, order=nil)
```

Comment:
Some big methods in this model, could they be broken down a bit?

Reply:
I have broken them down a bit now.